lib/item.rb

```diff
@@ -1,3 +1,11 @@
 class Item
+    
+    attr_reader :name,
+                :price
+
+    def initialize(attributes)
+        @name = attributes[:name]
+        @price = attributes[:price]
```

Comment:
The interaction patterns asks to clean the string (remove the $) and convert to a float.